src/components/UserProfile/TeamsAndProjects/AddTeamPopup.jsx

```diff
@@ -18,18 +31,37 @@ const AddTeamPopup = React.memo(props => {
       onValidation(false);
     }
   };
+
   const selectTeam = team => {
     onSelectTeam(team);
     onValidation(true);
   };
 
+  const onCreateTeam = async () => {
```

Comment:
You should probably add another case in this function to check if there already exists a team with the given name.